samples/matter/common/src/app/matter_init.cpp

```diff
@@ -147,6 +148,36 @@ CHIP_ERROR ConfigureThreadRole()
 
 	return ConnectivityMgr().SetThreadDeviceType(threadRole);
 }
+
+CHIP_ERROR InitNetworkingStack()
+{
+	CHIP_ERROR error{ CHIP_NO_ERROR };
+
+	error = ThreadStackMgr().InitThreadStack();
+	VerifyOrReturnLogError(error == CHIP_NO_ERROR, error);
+
+	error = ConfigureThreadRole();
+	VerifyOrReturnLogError(error == CHIP_NO_ERROR, error);
+
+	return error;
+}
+
+#elif defined(CONFIG_CHIP_WIFI)
+
+CHIP_ERROR InitNetworkingStack()
+{
+	CHIP_ERROR error{ CHIP_NO_ERROR };
+
+	if (!sLocalInitData.mNetworkingInstance) {
```

Comment:
Is this check necessary? If someone wants to postpone the network commissioning instance for whatever reason, that should be OK, right? Especially that this defaults to a non-null instance, so a user can't miss this setting unintentionally.

Reply:
Would it be ok to keep the check before calling the Init() but do not report an error if the instance is NULL?

Reply:
Yes, this is what I meant :)